helpers/addresses.py

```diff
@@ -267,7 +267,7 @@
         "StrategyCvxHelper": "0x9A12A9141363A5B343B781c4951d42E327B89397", # V1.1
         "StrategyCvxCrvHelper": "0x76328277232c97BAf76D23A69015CB478293A048", # V1.1
         "KeeperAccessControl": "0x4fe70eE8fa906D59A88DE5946F114BdbFC410a80",
-        "native.vestedCVX": "0x56df16192D586b288F53852466560b445E21939b", # V1.4
+        "native.vestedCVX": "0xeA563B601C2Ce9c67a1f41222856E06AAB76D1fa", # V1.5
```

Comment:
verified source, deployed by the dao, v1.5 strat

Reply:
confirmed ops_deployer3 deployed and [diff](https://github.com/GalloDaSballo/vested-cvx/compare/cvx.1.4...cvx.1.5-rfc.1) introducing keeper methods